pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```diff
@@ -42,13 +40,6 @@ let
       NTFS3_LZX_XPRESS = yes;
       NTFS3_FS_POSIX_ACL = yes;
 
-      # Preemptive Full Tickless Kernel at 500Hz
-      SCHED_CORE = lib.mkForce (option no);
-      PREEMPT_VOLUNTARY = lib.mkForce no;
-      PREEMPT = lib.mkForce yes;
```

Comment:
The kernel common config would produce the opposite result with `PREEMPT_*` disabled.

Reply:
But `PREEMPT_VOLUNTARY` is set to y in upstream xanmod config,

Reply:
Ah, I see.